compiler/src/dotty/tools/dotc/core/Definitions.scala

```diff
@@ -338,6 +339,12 @@ class Definitions {
     def Predef_classOf(implicit ctx: Context) = Predef_classOfR.symbol
     lazy val Predef_undefinedR = ScalaPredefModule.requiredMethodRef("???")
     def Predef_undefined(implicit ctx: Context) = Predef_undefinedR.symbol
+    // The set of all wrap{X, Ref}Array methods, where X is a value type
+    lazy val Predef_wrapArrayR: Set[TermRef] = {
+      val methodNames = ScalaValueTypes.map(TreeGen.wrapArrayMethodName) + nme.wrapRefArray
+      methodNames.map(ScalaPredefModule.requiredMethodRef).toSet
+    }
+    def Predef_wrapArray(implicit ctx: Context): Set[Symbol] = Predef_wrapArrayR.map(_.symbol)
```

Comment:
To avoid redoing the `.map(_.symbol)` every time this method is called it would make sense to cache its result. Symbols are valid at least for one run so you can use the `PerRun` class for this, grep for `PerRun` in this file for some examples.

Reply:
Done.